dashboard/src/components/ProvisionButton/index.tsx

```diff
@@ -116,13 +120,23 @@ class ProvisionButton extends React.Component<IProvisionButtonProps, IProvisionB
     });
   };
 
+  public handleBackButton = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    this.setState({ ...this.state, displayNameForm: true });
```

Comment:
do you need the `...this.state`?

Reply:
oops no, thanks for catching that, I was debugging something and left that in!